src/app/services/ci-dashboard.service.ts

```diff
@@ -36,4 +36,9 @@ export class DashboardData {
         const data = this.http.get<any[]>(GlobalConstants.apiURL() + `${endpoint}`);
         return data;
     }
+    getMayastorTest() {
+        const data = this.http.get<any[]>('https://staging.openebs.ci/mayastor/mayastor'); // hardcoded with staging URL
```

Comment:
Let's create a constant file from where we will be getting the host names. So that it will be more scalable and we have access to that same  data throughout the application.